### ISSUE TITLE
4.0.1 Won't Compile DuplicateRelatedFileException

### ISSUE THREAD

**Issue by niebloomj** [intercom/intercom-android#412]
Please use the following template to submit your issue. Following this template will allow us to quickly investigate and help you with your issue. Please be aware that issues which do not conform to this template may be closed.

For feature requests please contact us at team@intercom.io

## Version info
  - Intercom for Android version: 4.0.1
  - Android OS version: 26
  - Android Device: Pixel

## Expected behavior

Compiling

## Actual behavior

Not Compiling

## Steps to reproduce
 1. Adding to gradle File
 2. Gradle Sync

## Logs

All input files are considered out-of-date for incremental task ':app:transformResourcesWithMergeJavaResForDebug'.
:app:transformResourcesWithMergeJavaResForDebug FAILED
:app:transformResourcesWithMergeJavaResForDebug (Thread[Task worker for ':',5,main]) completed. Took 1.908 secs.
:app:buildInfoGeneratorDebug (Thread[Task worker for ':',5,main]) started.
:app:buildInfoGeneratorDebug
Putting task artifact state for task ':app:buildInfoGeneratorDebug' into context took 0.0 secs.
Executing task ':app:buildInfoGeneratorDebug' (up-to-date check took 0.0 secs) due to:
  Task has not declared any outputs.
:app:buildInfoGeneratorDebug (Thread[Task worker for ':',5,main]) completed. Took 0.004 secs.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:transformResourcesWithMergeJavaResForDebug'.
> More than one file was found with OS independent path 'publicsuffixes.gz'

* Try:
Run with --debug option to get more log output.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':app:transformResourcesWithMergeJavaResForDebug'.
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:100)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:70)
	at org.gradle.api.internal.tasks.execution.Sk

**Comment by bboyle18** [comment]
Hi @niebloomj 👋 
Thanks for getting in touch.

Can I confirm with you that this is only happening once you added Intercom to your Gradle file?
I don't see anything specific to Intercom's code in your crash log, so it would be great if you could provide a little more info please 👍 

Thanks

Brian

**Comment by jackcsk** [comment]
Same here. I get the error:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':HB2015N2:transformResourcesWithMergeJavaResForDevDebug'.
> com.android.build.api.transform.TransformException: com.android.builder.packaging.DuplicateFileException: Duplicate files copied in APK publicsuffixes.gz
  	File1: /Users/jack/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/okhttp/3.8.1/4d060ca3190df0eda4dc13415532a12e15ca5f11/okhttp-3.8.1.jar
  	File2: /Users/jack/.android/build-cache/4fbec17d1c65a9e6cf97c3549959ff9fd0244a62/output/jars/libs/repackaged_dependencies.jar
```
I believe it is caused by repackaged okhttp in the intercom library

**Comment by bboyle18** [comment]
Thanks for adding this info @jackcsk 👍 
We'll take a look at this.
Can I ask if you were experiencing this issue pre the 4.0.1 release? 

Thanks

Brian

**Comment by jackcsk** [comment]
I've rolled back to version 3.2.10 and it works

**Comment by bboyle18** [comment]
Thanks for the update @jackcsk. We'll look into this issue and get back to you

**Comment by Mathbl** [comment]
@jackcsk In the meantime, if you want to use 4+, you can add this in your build.gradle

```gradle
packagingOptions {
    pickFirst 'publicsuffixes.gz'
}
```

**Comment by bboyle18** [comment]
Thanks @Mathbl 👍

**Comment by niebloomj** [comment]
@Mathbl  will try that out!

**Comment by jackcsk** [comment]
Thanks @Mathbl, I can compile after adding the `pickFirst` line.

**Comment by ronocod** [comment]
This should be fixed in the next release.

**Comment by ronocod** [comment]
Fixed in 4.0.3
